src/mono/mono/mini/debugger-agent.c

```diff
@@ -6629,6 +6632,10 @@ vm_commands (int command, int id, guint8 *p, guint8 *end, Buffer *buf)
 		PRINT_DEBUG_MSG (1, "[dbg] Protocol version %d.%d, client protocol version %d.%d.\n", MAJOR_VERSION, MINOR_VERSION, major_version, minor_version);
 		break;
 	}
+	case MDBGPROT_CMD_VM_SET_CLIENT_POS_CONTEXT_SP: {
+		pos_rsp = decode_int (p, &p, end);
+		break;
+	}
```

Comment:
This isn't needed, I think. `pos_rsp` is not used in debugger-agent anymore.

Reply:
You are right, thanks!